### ISSUE TITLE
--newest takes no effect

### ISSUE THREAD

**Issue by vxna** [tjunnone/npm-check-updates#509]
- [x] I have read the list of [known issues](https://github.com/tjunnone/npm-check-updates#known-issues) before filing this issue.
- [x] I have searched for [similiar issues](https://github.com/tjunnone/npm-check-updates/issues) before filing this issue.

* node version: v11.10.1
* npm version: 6.7.0
* npm-check-updates version: 3.0.8

---------------------------

Reproduction:
```
$ yarn add react-scripts react-helmet-async
info Direct dependencies
├─ react-helmet-async@0.2.0
└─ react-scripts@2.1.8
$ ncu -n
Using package.json
[====================] 1/1 100%

All dependencies match the newest package versions :)
```

Newest `react-scripts` version to date is [3.0.0-next.b0cbf2ca](https://www.npmjs.com/package/react-scripts/v/3.0.0-next.b0cbf2ca), `react-helmet-async` is [1.0.0-beta.1](https://www.npmjs.com/package/react-helmet-async/v/1.0.0-beta.1)

**Comment by stoically** [comment]
ncu v3 filters prerelease versions by default. To get the desired behavior in this case you would need `ncu -n --pre`.

**Comment by vxna** [comment]
Thanks, but what is the rationale behind this change? It seems confusing and nonintuitive.

**Comment by stoically** [comment]
The [release notes](https://github.com/tjunnone/npm-check-updates/releases/tag/v3.0.0) have information on why this change was made. Personally I'd go with filtering prereleases for `latest` (the default), but don't filter for `newest` and `greatest` since an explicit choice is made in this case.

**Comment by vxna** [comment]
> Personally I'd go with filtering prereleases for `latest` (the default), but don't filter for `newest` and `greatest` since an explicit choice is made in this case.

Can agree, I am all for not filtering out the `newest` if it's safe. 
`ncu -n --pre` is confusing. What others think?

**Comment by raineorshine** [comment]
Totally agree. I have updated `--newest` and `--greatest` to enable `--pre` by default. (I had to change it to `--pre 1` so that it can be disabled via the same option.)

**Comment by raineorshine** [comment]
Published in v3.1.0.
